astroplan/plots/tests/test_sky.py

```diff
@@ -22,3 +22,21 @@ def test_image_example():
     ax = fig.add_subplot(1, 1, 1)
     ax.plot([1, 2, 3])
     return fig
+
+
+@pytest.mark.skipif('not HAS_MATPLOTLIB')
+@pytest.mark.mpl_image_compare
+def test_timezone():
+    import matplotlib.pyplot as plt
+    from astropy import coordinates
+    from astropy import units as u
+    from ..time_dependent import plot_airmass
+    from ... import Observer
+    import datetime
+    import pytz
+
+    bg = coordinates.SkyCoord.from_name('Betelgeuse')
```

Comment:
Could you please replace the name resolver class method with a constructor that uses hard-coded coordinates to avoid an unnecessary internet query?